Sources/Raygun_KSCrash/Recording/Monitors/Raygun_KSCrashMonitor.c

```diff
@@ -137,52 +137,52 @@ static inline void setMonitorEnabled(Monitor* monitor, bool isEnabled)
 
 static inline bool isMonitorEnabled(Monitor* monitor)
 {
-    KSCrashMonitorAPI* api = getAPI(monitor);
+    Raygun_KSCrashMonitorAPI* api = getAPI(monitor);
     if(api != NULL && api->isEnabled != NULL)
     {
         return api->isEnabled();
     }
     return false;
 }
 
-static inline void addContextualInfoToEvent(Monitor* monitor, struct KSCrash_MonitorContext* eventContext)
+static inline void addContextualInfoToEvent(Monitor* monitor, struct Raygun_KSCrash_MonitorContext* eventContext)
 {
-    KSCrashMonitorAPI* api = getAPI(monitor);
+    Raygun_KSCrashMonitorAPI* api = getAPI(monitor);
     if(api != NULL && api->addContextualInfoToEvent != NULL)
     {
         api->addContextualInfoToEvent(eventContext);
     }
 }
 
-void kscm_setEventCallback(void (*onEvent)(struct KSCrash_MonitorContext* monitorContext))
+void raygun_kscm_setEventCallback(void (*onEvent)(struct Raygun_KSCrash_MonitorContext* monitorContext))
 {
     g_onExceptionEvent = onEvent;
 }
 
-void kscm_setActiveMonitors(KSCrashMonitorType monitorTypes)
+void raygun_kscm_setActiveMonitors(Raygun_KSCrashMonitorType monitorTypes)
 {
-    if(ksdebug_isBeingTraced() && (monitorTypes & KSCrashMonitorTypeDebuggerUnsafe))
+    if(raygun_ksdebug_isBeingTraced() && (monitorTypes & Raygun_KSCrashMonitorTypeDebuggerUnsafe))
     {
         static bool hasWarned = false;
         if(!hasWarned)
         {
             hasWarned = true;
-            KSLOGBASIC_WARN("    ************************ Crash Handler Notice ************************");
-            KSLOGBASIC_WARN("    *     App is running in a debugger. Masking out unsafe monitors.     *");
-            KSLOGBASIC_WARN("    * This means that most crashes WILL NOT BE RECORDED while debugging! *");
-            KSLOGBASIC_WARN("    **********************************************************************");
+            RAYGUN_KSLOGBASIC_WARN("    ************************ Crash Handler Notice ************************");
+            RAYGUN_KSLOGBASIC_WARN("    *     App is running in a debugger. Masking out unsafe monitors.     *");
+            RAYGUN_KSLOGBASIC_WARN("    * This means that most crashes WILL NOT BE RECORDED while debugging! *");
+            RAYGUN_KSLOGBASIC_WARN("    **********************************************************************");
         }
-        monitorTypes &= KSCrashMonitorTypeDebuggerSafe;
+        monitorTypes &= Raygun_KSCrashMonitorTypeDebuggerSafe;
     }
-    if(g_requiresAsyncSafety && (monitorTypes & KSCrashMonitorTypeAsyncUnsafe))
+    if(g_requiresAsyncSafety && (monitorTypes & Raygun_KSCrashMonitorTypeAsyncUnsafe))
     {
-        KSLOG_DEBUG("Async-safe environment detected. Masking out unsafe monitors.");
-        monitorTypes &= KSCrashMonitorTypeAsyncSafe;
+        RAYGUN_KSLOG_ERROR("Async-safe environment detected. Masking out unsafe monitors.");
```

Comment:
I see several log statements here that have changed from DEBUG/INFO to ERROR, are all these intentional?

Reply:
They weren't intentional. Must of been a quick find and replace mistake. I have gone back though and reverted them.

Reply:
Thanks, I'll continue my review